group_vars/wingedhelix_cluster/vars.yml

```diff
@@ -41,7 +41,6 @@ slurm_qos_limit_fractions:
   interactive-short:
     user: 0.5
 repo_manager: 'pulp'
-os_distribution: 'centos7'
```

Comment:
WH has removed `os_distribution` but in contrast of the CF (I haven't checked the others) it is not defined in the static inventories

Reply:
Relocated to ```group_vars/wingedhelix_cluster/pulp.yml``` similar to Vaxtron.

Reply:
Huh? OS should be defined in group vars or static inventory - as normally expected.
And then pulp should respect those values.

Reply:
I put it in ```group_vars/wingedhelix_cluster/pulp.yml ```, because we currently have a default configured in ```roles/pulp_server/defaults/main.yml```, but that ```os_distribution``` is used in other roles/places too, so maybe it should get relocated...

Reply:
But then they should not be named  the same... and pulp should have own
```
if pulp_os_distribution is not defined; then
pulp_os = os_distribution
```

Reply:
I'll move ```os_distribution``` back to ```group_vars/stack_name/main.yml``` for now. Relocating the default and or refactoring variable names is something for a new PR.